resim/third_party/ros2/copts.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ros2/cc_opts.bzl b/ros2/cc_opts.bzl
+index 4af74b8..3b50ea9 100644
+--- a/ros2/cc_opts.bzl
++++ b/ros2/cc_opts.bzl
+@@ -1,5 +1,5 @@
+ """ Defines commonly used C/C++ options.
+ """
+ 
+-C_COPTS = ["-std=c11"]
+-CPP_COPTS = ["-std=c++17"]
++C_COPTS = []
++CPP_COPTS = []
```

Comment:
We need this patch to use C++20 when building ROS2 targets.

Reply:
I'm about 90% confident this comment is obscuring a bunch more complexity, but I'll believe you.